cluster/terraform_kubernetes/variables.tf

```diff
@@ -182,3 +182,36 @@ locals {
     var.environment     # cluster1, cluster2, etc
   )
 }
+
+# Define variables for namespace, key vault, slack channel, and resource configurations
+variable "namespace" {
+  default = "monitoring"
+}
+
+variable "key_vault_id" {
+  description = "The Azure Key Vault ID where secrets are stored"
+}
+
+variable "slack_channel" {
+  default = "#teacher-services-infra"
+}
+
+variable "alertmanager_image_version" {
+  default = "v0.19.0"
+}
+
+variable "alertmanager_cpu_limit" {
+  default = "1"
+}
+
+variable "alertmanager_memory_limit" {
+  default = "1Gi"
+}
+
+variable "alertmanager_cpu_request" {
+  default = "500m"
+}
+
+variable "alertmanager_memory_request" {
+  default = "500M"
+}
```

Comment:
we normally have just alertmanager_app_mem and alertmanager_app_cpu. In this case mem would be 1Gi for both limit & request. Cpu limit would be hardcoded to 1, and cpu_request would use the var. Make as small as possible in the default, and it would be overridden in tfvars for test and prod

Reply:
fixed